### PR TITLE
docs: mark shell commands explicitly

### PR DESCRIPTION
**Pull request by Chris53897**
this change will add some syntax highlighting for some ide like PHPStorm and make them runable by a simple click.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,21 @@ Symfony and API Platform.
 
 Install composer dependencies:
 
-```
+```sh
```
Comment (dunglas) [inline README.md:12]:
```suggestion
```console
```
Console will be even better: https://stackoverflow.com/a/49817279

Comment (Chris53897) [reply README.md]: I changed it to `console`
But now PhpStorm does not recognize it.
I guess i can open an issue on PhpStorm.

**Comment by dunglas** [comment]
Thanks.

**Comment by Chris53897** [comment]
https://youtrack.jetbrains.com/issue/WI-74002/Markdown-Detect-keyword-console-for-one-click-runners

**Pull request merged**